solver/src/core/state/domain.rs

```diff
@@ -91,3 +99,15 @@ impl FixedDomain {
         (self.num.ub as f32) / (self.denom as f32)
     }
 }
+
+impl Display for FixedDomain {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.is_bound() {
+            write!(f, "{:.3}", self.lb())
+        } else if self.is_empty() {
+            write!(f, "none")
```

Comment:
∅
You are allowed to print utf8 characters if they are not uncommon.

Reply:
Thanks for the tip!